var/spack/repos/builtin/packages/rust-bootstrap/package.py

```diff
@@ -45,6 +67,17 @@ class RustBootstrap(Package):
                 "powerpc64le": "c5aedb12c552daa18072e386697205fb7b91cef1e8791fe6fb74834723851388",
             },
         },
+        "1.77.0": {
+            "darwin": {
+                "x86_64": "06a8e3b00248b80bc70c859a0fc652f337d53cff11dab49e7f07db4206d28ae7",
+                "aarch64": "8a0c1b5344acfc91cf12351cc5668b2fc0ff2e96b69aba1510c9dedf86185988",
+            },
+            "linux": {
+                "x86_64": "cbdd1b93046ea4aed974016e328cb09ad0b8ee96893bd5d548fd3e7907607c75",
+                "aarch64": "df4630b250c53227649214fa5d57240217686e86bf502e5de0402d3264cb74d8",
+                "powerpc64le": "8593c41db9f01688ea290286aa2910462c91879ec23a6faece1a682b69436651",
+            },
+        },
```

Comment:
This should be removed since no versions of `rust` directly require `rust-bootstrap@1.77.0`.